pandas/tests/formats/test_style.py

```diff
@@ -8,11 +8,6 @@
 from pandas.util.testing import TestCase
 import pandas.util.testing as tm
 
-# this is a mess. Getting failures on a python 2.7 build with
-# whenever we try to import jinja, whether it's installed or not.
-# so we're explicitly skipping that one *before* we try to import
-# jinja. We still need to export the imports as globals,
-# since importing Styler tries to import jinja2.
 job_name = os.environ.get('JOB_NAME', None)
```

Comment:
Oops, meant to just delete the first sentence. I don't know if it's still a problem, will test it out in a followup.